### ISSUE TITLE
How to remap multiple keys?

### ISSUE THREAD

**Issue by Leoongithub** [ililim/dual-key-remap#5]
Can I use this to modify more than one key at the same time?

**Comment by ililim** [comment]
Currently that's not implemented, what workflow are you trying to implement? For general purpose remapping I recommend https://www.autohotkey.com/.

Adding multiple key support is a good idea, but implementing it would take some extra work and isn't planned yet in the upcoming future. I'd be happy to accept code contributions in this area.

**Comment by Leoongithub** [comment]
I'm not sure if I express it clearly. I mean, for example:
```
{remap_key=CAPITAL
when_alone=ESCAPE
with_other=CTRL}
{remap_key=ENTER
when_alone=ENTER
with_other=CTRL}
````

**Comment by ililim** [comment]
Yep, I understand. This isn't possible yet, but I'll add it to the list for future features.

**Comment by ShiranC** [comment]
I would like to do the same thing @Leoongithub requested. It would be great to have this feature!
Thanks @ililim for the app.

**Comment by ililim** [comment]
This has been implemented in the [latest release (v0.4)](https://github.com/ililim/dual-key-remap/releases). For example, the above remapping can be achieved with the following config:
```
remap_key=CAPSLOCK
when_alone=ESCAPE
with_other=CTRL

remap_key=ENTER
when_alone=ENTER
with_other=CTRL
```
